scripts/cnv_wdl/germline/cnv_germline_case_workflow.wdl

```diff
@@ -314,7 +326,7 @@ task DetermineGermlineContigPloidyCaseMode {
             --mapping-error-rate ~{default="0.01" mapping_error_rate} \
             --sample-psi-scale ~{default="0.0001" sample_psi_scale}
 
-        tar czf case-contig-ploidy-calls.tar.gz -C ~{output_dir_}/case-calls .
+        tar c -C ~{output_dir_}/case-calls . | gzip -1 > case-contig-ploidy-calls.tar.gz
```

Comment:
Can you explain this change? Also, why is it not in cohort mode as well?

Reply:
Faster compression with `gzip -1` I believe. This is okay in case mode since the calls tars aren't usually kept in storage except as intermediates, so the tradeoff with larger file size doesn't outweigh the cost of compressing/decompressing on VMs.